pom.xml

```diff
@@ -15,7 +15,7 @@
     </modules>
 
     <properties>
-        <revision>7.1.0</revision>
+        <revision>8.0.0</revision>
```

Comment:
Should this change not be part of the Release process?

Reply:
this pull request introduces breaking changes